CHANGELOG.md

```diff
@@ -13,4 +16,4 @@
 
 ### Release 1.0.0-rc.1
 
-* Initial preview release of the QLDB Python Sample Application.
\ No newline at end of file
+* Initial preview release of the QLDB Python Sample Application.
```

Comment:
Is there a whitespace character change here?

Reply:
No - added no change here: just added at the top.
I tried hard reverting to previous commit and just adding from the top again in vim, same thing showed up in git diff.